R/locations_and_mascots.R

```diff
@@ -0,0 +1,152 @@
+## LOCATIONS AND MASCOTS (by: pep)
```

Comment:
I'm not sure what is happening with the `NAMESPACE` file. We don't want all those deletions.

Reply:
I haven't used the NAMESPACE file though.

Reply:
The only major deletions I made was when I created an .Rmd file to test code in (as to not mess with anything already written and working), and then deleted it after I knew it worked and was added to the appropriate original script.

Reply:
Maybe just unstaged `NAMESPACE` from the commit? Your code shouldn't be changing the `NAMESPACE` file at all.

Reply:
Also, this file should be in `data-raw/`, not in `R/`. Can you call it `data-raw/98_mascots.R`?

Reply:
Sounds good, fixed that and pushing it now. Also, running checks now... had to change the number of expected columns in the test-colors.R from 8 to 9 [since added the mix-matched sportslogos_name column]. Additionally, where can I find the "documentation entries" and the "code/documentation" so I can update that to match what I've added? I checked where I thought this would be, but I'm having trouble finding it. 

<img width="539" alt="Screen Shot 2019-11-25 at 8 47 09 AM" src="https://user-images.githubusercontent.com/33033193/69550239-48bf6000-0f60-11ea-8330-4069442f45e3.png">

Reply:
It means that the variables in the data frame do not match what the documentation for the variables are in `R/data.R`. So you'll have to change that.

Reply:
Also, I don't think we want `teamcolors_updated`. We just want `teamcolors` to be updated.

Reply:
Okay, sounds good. I wasn't sure if you wanted me replacing the original one or not. I'm going to have to leave soon to go take the LSAT, so I'll work on it more this afternoon and push the corrections this evening. I think I should be able to get the check to pass. I'll let you know if anything else comes up.

Reply:
I'm so close.

<img width="579" alt="Screen Shot 2019-11-26 at 10 29 11 AM" src="https://user-images.githubusercontent.com/33033193/69652730-bbe7d580-1037-11ea-8135-2bf44ef681a4.png">

I found where this is referring to... but I'm not sure what I'm suppose to do. I haven't really done anything with the colors. And the files that the error is referencing are the teamcolors.Rd and league_pal.Rd that are in the man folder. Additionally, they are files that are read only.

Reply:
OK, let me take it from here. Thanks @pep93!

Reply:
No problem @beanumber! Please let me know if there's anything else I should be working on!